credoai/utils/lens_utils.py

```diff
@@ -0,0 +1,12 @@
+def get_metric_keys(module_prepared_results, lens_prepared_results):
+    """
+    Connects prepared results from a module to metric keys generated by Lens
+
+    Parameters
+    ----------
+    module_prepared_results : DataFrame
+        output of CredoModule.prepare_results()
+    lens_prepared_results : DataFrame
+        output of lens._prepare_results(assessment)
+    """
+    return module_prepared_results.merge(lens_prepared_results)['metric_key'].tolist()
```

Comment:
Isn't this missing `on` parameter? Column name to join on.

Reply:
it's automatically "on" all overlapping columns. But I've moved away from this function anyway